packages/patternfly-4/react-core/src/components/Wizard/index.ts

```diff
@@ -1,8 +1,8 @@
-export { Wizard } from './Wizard';
+export * from './Wizard';
```

Comment:
This will still work without the `export default` in Wizard*.tsx.

Reply:
yep updated it.